vignettes/statistical_power.Rmd

```diff
@@ -0,0 +1,161 @@
+  
+  4. *Statistical power*: This brings us to statistical power, which can be thought of in many ways, such as the probability that we are *correctly* observing an effect or group difference, or that we are correctly rejecting the null hypothesis, and so on (see, e.g., [@cohen1988], [@greene2000] for more). But regardless of the interpretation, all of these interpretations are all pointing to a common idea: *the ability for us to trust the result we get from the hypothesis test*, regardless of the test. 
+  
+Let's put these pieces together with a simple example. Say we find a "statistically significant" ($p < 0.05$) difference between two group means from a two-sample t-test. In this case, we might be tempted to stop and conclude that the signal is sufficiently strong to conclude that the groups are different from each other. But our test could be incorrect for a variety of reasons. Recall, that the p-value is a *probability*, meaning in part that we could be erroneously rejecting the null hypothesis, or that an insignificant result is insignificant due to a small sample size, and so on. 
+
+> This is where statistical power comes in.
+
+Statistical power helps us go the next step and more thoroughly assess the probability that the "significant" result we observed is indeed significant, or detect a cause of an insignificant result (e.g., sample size). In general, *before* beginning a broader analysis, it is a good idea to check for statistical power to ensure that you can trust the results you get from your test(s) downstream, and that your inferences are reliable. 
+  
+So this is where we focus in this vignette, and pay special attention to the ease and role of effect size calculation via the `effectsize` package from easystats. The following section walks through a simple applied example to ensure 1) the concepts surrounding and involved in power are clear and digestible, and 2) that the role and value of the `effectsize` package are likewise clear and digestible. Understanding both of these realities will allow for more complex extensions and applications to a wide array of research problems and questions. 
+
+# An Applied Example
+
+In addition to relying on the easystats `effectsize` package for effect size calculation, we will also leverage the simple, but excellent `pwr` package for the following implementation of power analysis [@champley2017]. 
+
+```{r}
+library(effectsize)
+library(pwr)
+```
+
+First, let's fit a simple two sample t-test using the mtcars data to explore mean MPG for both transmission groups (`AM`).
+
+```{r}
+t <- t.test(mpg ~ am, data = mtcars)
+```
+
+There are many power tests supported by `pwr` for different contexts, and we encourage you to take a look and select the appropriate one for your application. For present purposes of calculating statistical power for our t-test, we will rely on the `pwr.t.test()` function. Here's the basic anatomy:
+
+```{r eval = FALSE}
+pwr.t.test(d = ..., 
+           n = ..., 
```

Comment:
@pdwaggoner can you fix the linting issues? It'll be easier to review

Reply:
@DominiqueMakowski - Thanks! With some great help from @IndrajeetPatil , I have addressed all issues I am able to address. There are a few remaining items that are dependency issues with other easystats packages, but nothing at this point should be affecting the new vignette. Not sure if you want to "override" these issues. They seem strict. But let me know if there's something I am missing or can correct.